tests/testament/categories.nim

```diff
@@ -522,6 +522,8 @@ proc processCategory(r: var TResults, cat: Category, options: string) =
   of "untestable":
     # We can't test it because it depends on a third party.
     discard # TODO: Move untestable tests to someplace else, i.e. nimble repo.
-  else:
```

Comment:
else:
  let dir = "tests" / .? cat.string
  doAssert os.existsDir(dir), dir
  for name in os.walkFiles(dir / "t*.nim"): ...